timeless/auth/views.py

```diff
@@ -75,7 +76,7 @@ def activate():
         return render_template("auth/activate.html")
```

Comment:
@emilianodellacasa 
https://github.com/timelesslounge/timelessis/pull/500/files#diff-f8e768bb9d834f80ce350320e14c6050R75 - according to this line, now I think it's wrong logic. 
`g` object lives during request. 
If we want to use session, we should do here smth like that - http://flask.pocoo.org/docs/1.0/quickstart/#sessions

Reply:
@vladarefiev So you are suggesting I should only used session?

Reply:
@emilianodellacasa 
`if not g.user:` is useless, and will never be equal to `False`

As I can see here https://github.com/timelesslounge/timelessis/blob/master/timeless/auth/auth.py#L39

We should check `if not session['user_id']`. WDYT?